### DISCUSSION TITLE
auction house

### DISCUSSION THREAD

**Discussion by colue** [Zrips/CMI#2887] [Feature Requests]
add in an in game ah would be awsome
---
allows players to sell items.

most of us have to find other plugins to get an auction house for our players

**Comment by DevBD1** [comment]
I think it is not needed since there are plenty of AH plugins on the market.

**Comment by colue** [comment]
yes there are but how many of them other plugins are updated and dont have major glitches

**Comment by DevBD1** [comment]
Most of them, I can suggest you some free resources.

**Comment by colue** [comment]
sure if you know of any that really work and are updated
